@xen-orchestra/lite/src/components/form/FormCheckbox.vue

```diff
@@ -53,7 +54,17 @@ const value = useVModel(props, "modelValue", emit);
 const type = inject<"checkbox" | "radio" | "toggle">("inputType", "checkbox");
 const hasLabel = inject("hasLabel", false);
 const isLabelDisabled = inject("isLabelDisabled", ref(false));
-const icon = computed(() => (type === "checkbox" ? faCheck : faCircle));
+const icon = computed(() => {
+  if (type !== "checkbox") {
+    return faCircle;
+  }
+
+  if (value.value === undefined) {
+    return faMinus;
+  }
+
+  return faCheck;
+});
```

Comment:
Don't we also want to set the HTML checkbox's native `indeterminate` attribute to `true`?

Reply:
There is no such HTML attribute.

We could possibly consider using the `elm.indeterminate` JS property but since it's only for a visual feedback and the original checkbox is not displayed, I'm not sure it worth it.

What do you think?

Reply:
Yes, that's what I meant. If it's only for a native visual feedback, I don't think it's necessary. But I was wondering if it was also useful for accessibility.

Reply:
The property is not used for accessibility.

However, there is an `aria-checked` attribute for accessibility (which can take true/false/mixed value).

It makes me think that we have never really addressed the subject of accessibility. Maybe we should talk about this subject at the next stand-up?